entity-framework/core/modeling/relationships/one-to-many.md

```diff
@@ -0,0 +1,585 @@
+                    .IsRequired(false);
+            }
+-->
+[!code-csharp[OneToManyOptionalShadowFromPrincipal](../../../../samples/core/Modeling/Relationships/OneToMany.cs?name=OneToManyOptionalShadowFromPrincipal)]
+
+### One-to-many without navigation to principal
+
+<!--
+        // Principal (parent)
+        public class Blog
+        {
+            public int Id { get; set; }
+            public ICollection<Post> Posts { get; } = new List<Post>();  // Collection navigation containing dependents
+        }
+
+        // Dependent (child)
+        public class Post
+        {
+            public int Id { get; set; }
+            public int BlogId { get; set; }                              // Required foreign key property
+        }
+-->
+[!code-csharp[OneToManyRequiredNoNavigationToPrincipal](../../../../samples/core/Modeling/Relationships/OneToMany.cs?name=OneToManyRequiredNoNavigationToPrincipal)]
+
+For this example, the foreign key property has been re-introduced, but the navigation on the dependent has been removed.
+
+> [!TIP]
+> A relationship with only one navigation--one from dependent to principal or one from principal to dependent(s), but not both--is known as a unidirectional relationship.
+
+As before, this relationship is [discovered by convention](xref:core/modeling/relationships/conventions). For cases where the navigations, foreign key, or required/optional nature of the relationship are not discovered by convention, these things can be configured explicitly. For example:
+
+<!--
+            protected override void OnModelCreating(ModelBuilder modelBuilder)
+            {
+                modelBuilder.Entity<Blog>()
+                    .HasMany(e => e.Posts)
+                    .WithOne()
+                    .HasForeignKey(e => e.BlogId)
+                    .IsRequired();
+            }
+-->
+[!code-csharp[OneToManyRequiredFromPrincipalNoNavigationToPrincipal](../../../../samples/core/Modeling/Relationships/OneToMany.cs?name=OneToManyRequiredFromPrincipalNoNavigationToPrincipal)]
+
+Notice that the call to `WithOne` has no arguments. This is the way to tell EF that there is no navigation from `Post` to `Blog`.
+
+If configuration starts from the entity with no navigation, then the type of the entity on the other end of the relationship must be explicitly specified using the generic `HasOne<>()` call. For example:
+
+<!--
+            protected override void OnModelCreating(ModelBuilder modelBuilder)
+            {
+                modelBuilder.Entity<Post>()
+                    .HasOne<Blog>()
+                    .WithMany(e => e.Posts)
+                    .HasForeignKey(e => e.BlogId)
+                    .IsRequired();
+            }
+-->
+[!code-csharp[OneToManyRequiredFromDependentNoNavigationToPrincipal](../../../../samples/core/Modeling/Relationships/OneToMany.cs?name=OneToManyRequiredFromDependentNoNavigationToPrincipal)]
+
+### One-to-many without navigation to principal and with shadow foreign key
```

Comment:
Not understanding how do correctly do unidirectional navigations is a common problem. I think this is a case that warrants multiple examples, so that people don't have to extrapolate to their own case.